external/openembedded-layer/recipes-extended/haveged/haveged_%.bbappend

```diff
@@ -1,14 +1,14 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 inherit ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'systemd', '', d)}
 
-SYSTEMD_PACKAGES = "${PN}"
-SYSTEMD_SERVICE:${PN} = "haveged.service"
+SYSTEMD_PACKAGES:tegra = "${PN}"
+SYSTEMD_SERVICE:${PN}:tegra = "haveged.service"
 
 # Based on init.d/service.redhat from 1.9.14
 # https://raw.githubusercontent.com/jirka-h/haveged/v1.9.14/init.d/service.redhat
-SRC_URI += "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'file://haveged.service', '', d)}"
+SRC_URI:tegra += "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'file://haveged.service', '', d)}"
```

Comment:
This should be `SRC_URI:append:tegra =`, and a blank will be needed at the front of the string.

Reply:
right, I will change it to `':append:tegra = " '`